feat: add a `with-tailwind` example (#359)
tuono-labs/tuono: apps/documentation/src/routes/documentation/styles/tailwind-css.mdx

# Tailwind CSS

> [Tailwind CSS](https://tailwindcss.com) is a popular utility-first CSS framework. Tuono support starts from Tailwind CSS v4, and you can install our example with `tuono new [NAME] --template with-tailwind.

Reviewer: ```suggestion
[Tailwind CSS](https://tailwindcss.com) is a popular utility-first CSS framework. Tuono support starts from Tailwind CSS v4.

You can start using right away using `with-tailwind` template:

\```sh
tuono new tuono-with-tailwind --template with-tailwind
\```
```

> [Tailwind CSS](https://tailwindcss.com) is a popular utility-first CSS framework. Tuono support starts from Tailwind CSS v4, and Tuono provides an [example](https://github.com/tuono-labs/tuono/tree/main/examples/with-tailwind) for ease of setup.

Reviewer 2: ```suggestion
[Tailwind CSS](https://tailwindcss.com) is a popular utility-first CSS framework. Tuono support starts from Tailwind CSS v4, you can install it with `tuono new [NAME] --template with-tailwind.
```
